src/Helsenorge.Messaging/Security/SignThenEncryptMessageProtection.cs

```diff
@@ -266,20 +266,20 @@ private byte[] Unprotect(byte[] data, X509Certificate2 signingCertificate)
             // Return the raw content (without a signature).
             return signedCms.ContentInfo.Content;
         }
-        
+
         private EnvelopedCms GetEnvelope(byte[] rawContent)
```

Comment:
I think we should add a test for this method as well.

And drop the braces when the if-statements just contain one line.